pin.go

```diff
@@ -0,0 +1,8 @@
+package drivers
+
+type Pin interface {
+	Get() bool
+	High()
+	Low()
+	Set(high bool)
```

Comment:
Maybe `Set(value bool)` instead?

Reply:
Looks like different boards have a different name there.

https://github.com/tinygo-org/tinygo/blob/8c54e3dd885824638015f319907f75c01570cdad/src/machine/machine_esp8266.go#L141
```go
func (p Pin) Set(value bool) {
```

https://github.com/tinygo-org/tinygo/blob/8c54e3dd885824638015f319907f75c01570cdad/src/machine/machine_nrf.go#L70
```go
func (p Pin) Set(high bool) {
```

I feel like `high` makes sense since a `true` value is high there, but it's not a strong opinion.

Reply:
Can't this function be used to set false value?

Reply:
It can, I see your point.